src/standfirst/__tests__/standfirst.stories.tsx

```diff
@@ -1,105 +1,154 @@
-export const StoryWithAsProp = () => (
-  <>
-    <StorybookSubHeading>Standfirst - as h3</StorybookSubHeading>
-    <Standfirst as="h3">{bodyString}</Standfirst>
-
-    <StorybookSubHeading>Standfirst - as span</StorybookSubHeading>
-    <Standfirst as="span">{bodyString}</Standfirst>
-  </>
-);
-StoryWithAsProp.storyName = 'with-as-prop';
-
-export const StoryWithOverrides = () => (
-  <>
-    <ThemeProvider theme={myCustomTheme}>
-      <StorybookSubHeading>
-        Standfirst - with style-preset override
-      </StorybookSubHeading>
+  <StorybookPage columns="1fr">
+    <StorybookCase>
       <Standfirst
+        as="h2"
         overrides={{
           styledText: {
-            stylePreset: 'standfirstCustom',
+            typographyPreset: 'utilitySubheading020',
           },
         }}
       >
         {bodyString}
       </Standfirst>
-      <StorybookSubHeading>
-        Standfirst - with typography-preset override
-      </StorybookSubHeading>
+    </StorybookCase>
+  </StorybookPage>
+);
+StoryDefault.storyName = 'Default';
+
+export const StoryWithAsSpan = () => (
+  <StorybookPage columns="1fr">
+    <StorybookCase>
       <Standfirst
+        as="span"
         overrides={{
           styledText: {
-            typographyPreset: 'editorialSubheadline020',
+            typographyPreset: 'utilitySubheading020',
           },
         }}
       >
         {bodyString}
       </Standfirst>
-    </ThemeProvider>
-  </>
+    </StorybookCase>
+  </StorybookPage>
 );
-StoryWithOverrides.storyName = 'with-overrides';
+StoryWithAsSpan.storyName = 'Standfirst as span';
 
 export const StoryLogicalProps = () => (
   <>
-    <StorybookSubHeading>
-      Standfirst - logical padding props
-    </StorybookSubHeading>
-    <Standfirst
-      overrides={{styledText: {paddingInline: '20px', paddingBlock: '10px'}}}
-    >
-      {bodyString}
-    </Standfirst>
-    <StorybookSubHeading>Standfirst - logical margin props</StorybookSubHeading>
-    <Standfirst
-      overrides={{styledText: {marginInline: '20px', marginBlock: '10px'}}}
-    >
-      {bodyString}
-    </Standfirst>
+    <StorybookPage columns="1fr">
+      <StorybookCase>
+        <MarginOverridesWrapper>
+          <Standfirst
+            as="span"
+            overrides={{
+              styledText: {
+                typographyPreset: 'utilitySubheading020',
+                paddingInline: '20px',
+                paddingBlock: '10px',
+              },
+            }}
+          >
+            {bodyString}
+          </Standfirst>
+        </MarginOverridesWrapper>
+      </StorybookCase>
+    </StorybookPage>
+  </>
+);
+StoryLogicalProps.storyName = 'Logical props';
+
+export const StoryWithStylingOverrides = () => (
+  <>
+    <StorybookPage columns="1fr">
+      <StorybookCase>
+        <Standfirst
+          as="span"
+          overrides={{
+            styledText: {
+              typographyPreset: 'utilitySubheading020',
+              stylePreset: 'standfirstCustom',
```

Comment:
You can use directly `inkBrand010` as style-preset, not need to create a custom one, newskit automatically creates style presets for all colors tokens which start with `ink`
```suggestion
              stylePreset: 'inkBrand010',
```